aten/src/ATen/detail/XPUHooksInterface.h

```diff
@@ -55,6 +51,10 @@ struct TORCH_API XPUHooksInterface {
     return 0;
   }
 
+  virtual DeviceIndex current_device() const {
+    return -1;
+  }
```

Comment:
This seems related to device, not generator.

Reply:
This is used in `manual_seed` in `xpu.cpp` file.